spec/fabricators/external_upload_stub_fabricator.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+Fabricator(:external_upload_stub) do
+  created_by { Fabricate(:user) }
+  original_filename "test.txt"
+  key "path/to/s3/#{SecureRandom.hex(10)}/test.txt"
```

Comment:
Hmm I think we need a consistent length the the unique identifier here. `SecureRandom.hex`, `SecureRandom.hex(10)` and `SecureRandom.hex(20)` generate strings of different lengths.

Reply:
Good catch; this will be a UUID now anyway!